src/PointwiseFunctions/AnalyticData/GeneralRelativity/InterpolateFromSpec.hpp

```diff
@@ -82,18 +82,19 @@ tuples::tagged_tuple_from_typelist<Tags> interpolate_from_spec(
 #ifdef SPECTRE_DEBUG
       const auto component_name =
           tensor.component_name(tensor.get_tensor_index(component_i));
-      if constexpr (tensor.rank() == 1) {
+      if constexpr (std::decay_t<decltype(tensor)>::rank() == 1) {
```

Comment:
This didn't show up on Wheeler or CaltechHPC with GCC. Does clang not support this or something like that?

Reply:
I don't know what the standard says, but Clang tends to be right more often than GCC when it comes to standard C++.